android/src/test/java/com/magicleap/magicscript/utils/ExtensionsKtTest.kt

```diff
@@ -138,4 +139,140 @@ class ExtensionsKtTest {
         sum shouldEqual 0f
     }
 
+    @Test
+    fun `should add Vector3`() {
+        val vector1 = Vector3(1f, 2f, 3f)
+        val vector2 = Vector3(2f, 3f, 4f)
+
+        vector1 + vector2 shouldEqual Vector3.add(vector1, vector2)
```

Comment:
I think it would be better to check against raw value: shouldEqual `Vector3(3f, 5f, 7f)` - it will detect possible error in `Vector3` implementation.

Reply:
I'm not testing vector3 implementation